src/modules/75-cd/components/PipelineSteps/DeployServiceStep/DeployServiceStep.tsx

```diff
@@ -543,7 +543,7 @@ export const DeployServiceWidget: React.FC<DeployServiceProps> = ({
           serviceRef: getServiceRefSchema(getString)
         })}
       >
-        {formik => {
+        {(formik: any) => {
```

Comment:
Codacy has a fix for the issue: [Unexpected any. Specify a different type.](https://app.codacy.com/gh/harness/harness-core-ui/pullRequest?prid=9232293)
```suggestion
        {(formik: unknown) => {
```